broadlink/__init__.py

```diff
@@ -66,8 +66,8 @@ def gendevice(devtype, host, mac):
     # Look for the class associated to devtype in devices
     [device_class] = [dev for dev in devices if devtype in devices[dev]] or [None]
     if device_class is None:
-        return device(host=host, mac=mac, devtype=devtype)
-    return device_class(host=host, mac=mac, devtype=devtype)
+        return device(host, mac, devtype, name=name, cloud=cloud)
+    return device_class(host, mac, devtype, name=name, cloud=cloud)
```

Comment:
I changed something that I don't completely understand here. Why were we using kwargs? Backwards compatibility? In that case, let me know and I will keep the previous syntax. Otherwise, I don't see why not use args.